src/components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -54,7 +54,7 @@
 
       <Search v-bind.sync="searchQuery">
         <b-field>
-          <Pagination simple replace preserveScroll :total="total" v-model="currentValue" :per-page="first" />
+          <Pagination className="mb-0" hasMagicBtn simple replace preserveScroll :total="total" v-model="currentValue" :per-page="first" />
```

Comment:
you don't need to create a className @prop as ```class=""``` will work

Reply:
@roiLeo it throws an error though that you cant use keywords as props